x-pack/platform/plugins/shared/onechat/public/application/components/conversations/new_conversation_button.tsx

```diff
@@ -27,12 +29,17 @@ export const NewConversationButton: React.FC<{}> = () => {
     if (isNewConversation) {
       cleanConversation();
     }
+    if (isEmbeddedContext) {
+      setConversationId?.(undefined);
+    }
   };
 
   const buttonProps = isDisabled
     ? {
         disabled: true,
       }
+    : isEmbeddedContext
```

Comment:
nit: chained ternary operators.... maybe let's make it `const getButtonProps = () => { if(isEmbeddedContext) { .....} ...}` for better readability?